src/Resources/views/storefront/page/product-detail/tabs.html.twig

```diff
@@ -0,0 +1,26 @@
+{% sw_extends '@Parent/storefront/page/product-detail/tabs.html.twig' %}
```

Comment:
Can we move the stylings to external file please?
Inline stylings are rather deprecated :)